src/main/java/group1/mips_simulator/simulator/README.md

```diff
@@ -0,0 +1 @@
+## UI of Simulator
```

Comment:
What is the plan with these README.md files? When will they be filled out?

Reply:
Oh I see, this is just a hold over from the other MR. See my comment on that MR then :)

Reply:
I'm considering them to be places where anyone can put notes/todo lists/design details that may not make it as part of a commit message; just thinking it would make things easier to put together in a Design doc when we make our submission, any notes could be extracted from these intermediate readmes

Alternatively, we'll need to write instructions (specifically for the UI). We can use these .md readmes as the basis for those.

thats what i was thinking at least, got any thoughts?